.travis.yml

```diff
@@ -10,6 +10,7 @@ python:
 cache: pip
 
 before_install:
+  - sudo add-apt-repository -y ppa:jonathonf/texlive-2016
```

Comment:
First I thought the origin of errors is the old (2013) version of texlive.
Then it was left there to use modern software.
What is your texlive version?